pluto-plugins/plugins/network/core/lib/src/main/java/com/pluto/plugins/network/internal/interceptor/ui/DetailsFragment.kt

```diff
@@ -24,8 +24,9 @@ import com.pluto.utilities.share.Shareable
 import com.pluto.utilities.share.lazyContentSharer
 import com.pluto.utilities.spannable.setSpan
 import com.pluto.utilities.viewBinding
+import io.ktor.http.Url
 
```

Comment:
The reason I used the Ktor url here is that it is multiplatform. not sure is this should be avoided since this dependency will go to people not using ktor as well :/.

Reply:
this is ok, 
this is part of core module.